WebContent/WEB-INF/jsp/eventHandlers.jsp

```diff
@@ -456,36 +456,43 @@
         var xid = $get("xid");
         var alias = $get("alias");
         var disabled = $get("disabled");
-        if (handlerType == <c:out value="<%= EventHandlerVO.TYPE_EMAIL %>"/>) {
+        var messages = [];
+        if (!(isNotEmpty(xid)) && handlerId !== ${NEW_ID}){
```

Comment:
`!(isNotEmpty(xid))` - Instead of so many negations, it is worth creating the isEmpty method.